skimage/morphology/misc.py

```diff
@@ -124,3 +127,81 @@ def remove_small_objects(ar, min_size=64, connectivity=1, in_place=False):
     out[too_small_mask] = 0
 
     return out
+
+def remove_small_holes(ar, min_size=64, connectivity=1, in_place=False):
+    """Remove connected components smaller than the specified size within a 
+       larger connected object.
+
+    Parameters
+    ----------
+    ar : ndarray (arbitrary shape, int or bool type)
+        The array containing the connected components of interest. If the array
+        type is int, it is assumed that it contains already-labeled objects. The
+        labels are not kept in the output image (this function always outputs
```

Comment:
Move this remark to a "Notes" section? (to keep the "Parameters" section concise enough).